test/ts/src/testCombinations.ts

```diff
@@ -0,0 +1,65 @@
+import {
```

Comment:
Perhaps a better filename here, something like `testRpcCombinations` to make it more explicit